ItemModCreationBoilerplate/Items/ExampleItem.cs

```diff
@@ -25,15 +22,12 @@ public class ExampleItem : ItemBase
 
         public override string ItemIconPath => "";
 
-        public override void Init(ConfigFile config)
+        internal override void Init(ConfigFile config)
```

Comment:
Missing an implemented/empty method "Initialization", which must be implemented by inheritors of ItemBase.